src/diffusers/models/unet_2d_condition.py

```diff
@@ -778,6 +778,7 @@ def forward(
         added_cond_kwargs: Optional[Dict[str, torch.Tensor]] = None,
         down_block_additional_residuals: Optional[Tuple[torch.Tensor]] = None,
         mid_block_additional_residual: Optional[torch.Tensor] = None,
+        down_intrablock_additional_residuals: Optional[Tuple[torch.Tensor]] = None,
```

Comment:
Ok for me! Think it's indeed a good idea to cleanly separate ControlNet and T2I here